### ISSUE TITLE
Trying to decode format of 'outputs' for translation to c_api

### ISSUE THREAD

**Issue by rtrahms** [zzh8829/yolov3-tf2#27]
Nice job on this project!  Works great in python on a model we custom trained.
I am in the process of using the TF c_api to translate to C/C++ for deployment.  I understand the input tensor (image, 416x416x3x1), but I am having a little trouble trying to figure out the format of the output tensor (for either YoloV3 or TinyYoloV3).  Referencing the last lines in those functions:

outputs = Lambda(lambda x: yolo_nms(x, anchors, masks, classes),
                     name='yolo_nms')((boxes_0[:3], boxes_1[:3]))

Pre-NMS the tensor shape would be similar to:
batch_size x 10647 x (num_classes + 5 bounding box attrs)

The number 10647 is equal to the sum 507 +2028 + 8112, which are the numbers of possible objects detected on each scale (for full YoloV3). The bbox values describing bounding box attributes stand for center_x, center_y, width, height, confidence.

So, looking for confirmation: if I used YoloV3Tiny with say 12 classes and two scales (not 3 like full YoloV3), the output tensor would look like the following: 

(507 + 2028) * 3 * (12 + 5) = 2535 * 3 * 60 = 456,300

Note: I am also consulting the combined_max_suppression API, which is the last step in the pb file I am loading:
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/image/combined_non_max_suppression
which defines the output as the following:

Returns:
'nmsed_boxes': A [batch_size, max_detections, 4] float32 tensor containing the non-max suppressed boxes. 'nmsed_scores': A [batch_size, max_detections] float32 tensor containing the scores for the boxes. 'nmsed_classes': A [batch_size, max_detections] float32 tensor containing the class for boxes. 'valid_detections': A [batch_size] int32 tensor indicating the number of valid detections per batch item. Only the top valid_detections[i] entries in nms_boxes[i], nms_scores[i] and nms_class[i] are valid. The rest of the entries are zero paddings.

Is this correct?  Any help would be appreciated.

Thanks!
Rob

**Comment by rtrahms** [comment]
I wrote some code to dump the sizes of both input and output tensors from this network (12 classes):

**Comment by rtrahms** [comment]
Confirmed this is the right output, closing this issue!

**Comment by AnaRhisT94** [comment]
@rtrahms Hey Rob, How did you train on a custom training set? I'm getting `nans` in the training.
